ui/pages/onboarding-flow/metametrics/metametrics.test.js

```diff
@@ -133,6 +134,7 @@ describe('Onboarding Metametrics Component', () => {
   });
 
   it('should render the Onboarding component when the current date is after the new privacy policy date', () => {
+    delete process.env.IN_TEST; // Unset this flag to test component
```

Comment:
This pattern is copied from an existing test.
https://github.com/MetaMask/metamask-extension/blob/develop/shared/modules/i18n.test.ts#L94

This flag is used in both e2e and unit tests.
We set this flag when setting up jest: https://github.com/MetaMask/metamask-extension/blob/develop/test/helpers/setup-helper.js#L7